docs/variants/msi_z690/community-test-results.md

```diff
@@ -0,0 +1,47 @@
+# Community test results - MSI Z690-A WIFI DDR4
+
+## About
+
+The following document contains information about additional tests which were
+carried out on the MSI Z690-A WIFI DDR4 platform with Dasharo firmware by
+community members.
+
+The following list is created based on the information from the
+community - people which are currently using the platform with Dasharo
+firmware.
+
+This document contains three main sections:
+- `Contributing` in which we are describing preferred of reporting the results.
+- `Testing method` in which we are describing our preferred testing method.
+- `Hardware Compatible List` in which in a labelled manner we show all
+    tests carried out by the community.
+- `Support us` - in which we describe the method of how it is possible to
+    contribute to the development of this list.
+
+## Contributing
+
+Feel free to perform different tests and report your results via
+[email](mailto:contact@dasharo.com) or submit a Pull Request to
+[Dasharo documentation repository](https://github.com/Dasharo/docs) or by using
+[Dasharo issues repository](https://github.com/Dasharo/dasharo-issues/issues).
+
+## Testing method
+
+Platform testing might be carried out on the basis of
+[Dasharo Test Specifiaction](../../unified-test-documentation/overview.md).
+
+If you prefer another mechanism for checking any functionality or check any
+mechanism that we did not include in the documentation - describe it in the
+notification.
+
+## Community test results
+
+| No.  | Description                                       | Link to description | Notes               |
+|:-----|:--------------------------------------------------|:-------------------:|:-------------------:|
+| 1.   |                                                   |                     |                     |
+
+## Support Us
+
+Hardware compatible with Dasharo firmware you might report by:
```

Comment:
If this is not a list then remove the dash:

```
Hardware compatible with Dasharo firmware you might report by
using `Dasharo Issues project`, which is available under this
[link](https://github.com/Dasharo/dasharo-issues/issues).
```

Reply:
I just delete this section, it overlapped in meaning with `Contributing` section.